app/forms/admin/move_advisers_form.rb

```diff
@@ -33,9 +33,9 @@ def subsidiaries
     private
 
     def destination_firm_fca_number_exists
-      unless subsidiaries.count > 0
-        errors.add(:destination_firm_fca_number, :does_not_exist, fca_number: destination_firm_fca_number)
-      end
+      errors.add(:destination_firm_fca_number,
+                 :does_not_exist,
+                 fca_number: destination_firm_fca_number) unless subsidiaries.count > 0
```

Comment:
My original thought, when I commented on this before but didn't actually specify, was:

```
return if subsidiaries.count > 0
errors.add(:destination_firm_fca_number,
                 :does_not_exist,
                 fca_number: destination_firm_fca_number)
```

which is what guard clause means to me (though i've never actually confirmed this is what they actually are)